lib/cli/src/commands/run.rs

```diff
@@ -585,8 +585,13 @@ impl ExecutableTarget {
                             );
                         }
 
-                        Module::new(&engine, &wasm)
-                            .with_context(|| format!("Unable to compile \"{}\"", path.display()))?
+                        let module = tracing::debug_span!("compiling_wasm")
```

Comment:
isn't there another way that having an object being built from a `tracing::debug_span!` command? That feel odd.